web-services/client/src/main/java/datawave/webservice/model/ModelList.java

```diff
@@ -13,13 +15,28 @@
 @XmlRootElement(name = "ModelList")
 public class ModelList extends BaseResponse implements Serializable, HtmlProvider {
     
+    private String jqueryUri;
+    private String dataTablesUri;
+    private String modelTableName;
+    
     private static final long serialVersionUID = 1L;
-    private static final String TITLE = "Model Names", EMPTY = "";
+    private static final String TITLE = "Model Names";
+    private static final String DATA_TABLES_TEMPLATE = "<script type=''text/javascript'' src=''{0}''></script>\n"
+                    + "<script type=''text/javascript'' src=''{1}''></script>\n"
+                    + "<script type=''text/javascript''>\n"
+                    + "$(document).ready(function() '{' $(''#myTable'').dataTable('{'\"bPaginate\": false, \"aaSorting\": [[0, \"asc\"]], \"bStateSave\": true'}') '}')\n"
+                    + "</script>\n";
+    
+    public ModelList(String jqueryUri, String datatablesUri, String dataWaveMetadataTableName) {
+        this.jqueryUri = jqueryUri;
+        this.dataTablesUri = datatablesUri;
+        this.modelTableName = dataWaveMetadataTableName;
```

Comment:
It seems my last comment on this was deleted. The parameter to ModelList should be named `modelTableName` not `dataWaveMetadataTableName`. It is the name of the model table. By default that happens to be the metadata table, but that might not always be the case. In other words, the function of that parameter is to supply the name of the table used for storing the model, not to supply the name of the datawave metadata table.